easybuild/easyconfigs/c/ChimPipe/ChimPipe-0.9.5-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'PackedBinary'
+
+name = 'ChimPipe'
+version = '0.9.5'
+versionsuffix = '-Python-2.7.12'
+
+homepage = 'https://%(namelower)s.readthedocs.org/'
+description = """ChimPipe is a computational method for the detection of novel transcription-induced
+ chimeric transcripts and fusion genes from Illumina Paired-End RNA-seq data. It combines junction
+ spanning and paired-end read information to accurately detect chimeric splice junctions at base-pair
+ resolution."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/Chimera-tools/%(name)s/archive']
+sources = ['v%(version)s.tar.gz']
+
+checksums = ['0ab55a0a38dd86845334125459b3961e11e427564ed14b22185c9cc17467c000']
+
+dependencies = [
+    ('BEDTools', '2.27.1'),
+    ('SAMtools', '1.5'),
+    ('BLAST+', '2.6.0', versionsuffix),
+]
+
+sanity_check_paths = {
+    'files': ['%(name)s.sh'],
+    'dirs': ['bin', 'src/awk', 'src/bash', 'tools'],
```

Comment:
Any specific binaries you can check for?

Reply:
@boegel I added a specific binary and added a folder inside of `bin`.